pkg/assets/assets.go

```diff
@@ -0,0 +1,37 @@
+package assets
+
+import (
+	"bytes"
+	"text/template"
+)
+
+// Important: Run "make generate" to regenerate code after modifying/adding/removing any asset
```

Comment:
Outdated information. It should be "make assets" for what I've seen in the Makefile.

Reply:
:+1: